project/apps/collaboration-service/src/main.ts

```diff
@@ -0,0 +1,36 @@
+import { NestFactory } from '@nestjs/core';
+import { MicroserviceOptions, Transport } from '@nestjs/microservices';
+import { WsAdapter } from '@nestjs/platform-ws';
+
+import { EnvService } from './env/env.service';
+import { CollaborationModule } from './collaboration.module';
+
+async function bootstrap() {
+  const appContext =
+    await NestFactory.createApplicationContext(CollaborationModule);
+  const envService = appContext.get(EnvService);
+  const NODE_ENV = envService.get('NODE_ENV');
+  const COLLABORATION_SERVICE_HOST = envService.get(
+    'COLLABORATION_SERVICE_HOST',
+  );
+  appContext.close();
```

Comment:
prob dw to do this similar to matching-service

Reply:
oops you're right, i missed out a commit while doing `git cherry-pick`.

amended.